src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1786,9 +1788,13 @@ void Partitioner::finalizeLinks() {
         } else {
             // A function call: find in the prototype subgraph
             auto& params = P.functions.at(sg_desc._funcall)._model->get_parameters();
+            std::cout << "SUBGRAPH PARAM: " << model->get_friendly_name() + "_" +
+                        sg_desc._funcall + "__" + std::to_string(subgr_idx_to - 1) + "_" + ptr->get_friendly_name() << std::endl;
             auto& proto = func_pipeline_type == FunctionPipelineType::CWAI
                               ? ptr  // no protos in the CWAI case..
-                              : all_functions.at(sg_desc._funcall).param_call_to_proto.at(ptr);
+                              : all_functions.at(sg_desc._funcall).param_call_to_proto.
+                                at(model->get_friendly_name() + "_" +
+                        sg_desc._funcall + "__" + std::to_string(subgr_idx_to - 1) + "_" + ptr->get_friendly_name());
```

Comment:
So shouldn't this just work?
```suggestion
                                at(CallParam{fcall, ptr});
```
where fcall is the pointer to a particular function call model?

Reply:
Fixed, thanks!